### PR TITLE
Fix/lazy loading and scroll after change channel

### PR DESCRIPTION
**Pull request by Dinek007**

### CODE REVIEW

**Comment by Dinek007** [comment]
> _ No description provided. _

**Review by holmesworcester — APPROVED** [review]

**Review by holmesworcester — APPROVED** [review]

File: packages/frontend/src/renderer/components/Channel/ChannelComponent.tsx [hunk 1]
```diff
@@ -71,11 +71,15 @@ export const ChannelComponent: React.FC<ChannelComponentProps> = ({
...
-  const { ref: scrollbarRef } = useResizeDetector({ onResize })
+  const { ref: scrollbarRef } = useResizeDetector<HTMLDivElement>({ onResize })
 
   const scrollBottom = () => {
     if (!scrollbarRef.current) return
-    scrollbarRef.current.scrollTop = scrollbarRef.current.scrollHeight
+    setScrollHeight(0)
+    scrollbarRef.current.scrollTo({
+      behavior: 'auto',
+      top: Math.abs(scrollbarRef.current?.clientHeight - scrollbarRef.current?.scrollHeight)
```
Comment (siepra) [inline packages/frontend/src/renderer/components/Channel/ChannelComponent.tsx:81]: Are you sure Math.abs is correct? Shouldn't it be `scrollbarRef.current?.clientHeight` and `scrollbarRef.current?.scrollHeight` both wrapped in `Math.floor`?

Comment (siepra) [reply packages/frontend/src/renderer/components/Channel/ChannelComponent.tsx:81]: I'd have checked it on Chromebook also

**Review by siepra — APPROVED** [review]
I'm approving those changes but I left a comment and also I wish you fixed checks on CI

File: packages/frontend/src/renderer/components/Channel/ChannelComponent.tsx (hunk 1 above)
Comment (Dinek007) [reply packages/frontend/src/renderer/components/Channel/ChannelComponent.tsx:81]: yes i want to convert to positive numbers

**Pull request merged**